src/main/java/com/theelixrlabs/healthcare/constants/MessageConstants.java

```diff
@@ -10,6 +10,7 @@ public class MessageConstants {
     public static final String AADHAAR_SHOULD_NOT_EMPTY = "doctor.aadhaarNumber.should.not.be.empty";
     public static final String DEPARTMENT_SHOULD_BE_MANDATORY = "{doctor.department.mandatory}";
     public static final String DOCTOR_ID_IS_MANDATORY = "{validation.doctorID.mandatory}";
+    public static final String DOCTOR_NAME_CANNOT_BE_EMPTY = "doctor.name.cannot.be.empty";
     public static final String DOCTOR_UNAVAILABLE = "doctor.unavailable";
     public static final String FIRST_NAME_SHOULD_BE_MANDATORY = "{doctor.firstName.mandatory}";
     public static final String FIRST_NAME_SHOULD_NOT_EMPTY = "doctor.firstName.should.not.be.empty";
```

Comment:
Rename FIRST_NAME_SHOULD_NOT_EMPTY to FIRST_NAME_SHOULD_NOT_BE_EMPTY

Reply:
changed to FIRST_NAME_SHOULD_NOT_BE_EMPTY